build-system/dep-check-config.js

```diff
@@ -50,12 +50,52 @@ exports.rules = [
           'third_party/closure-library/sha384-generated.js',
       'extensions/amp-mustache/0.1/amp-mustache.js->' +
           'third_party/mustache/mustache.js',
+      'extensions/amp-timeago/0.1/amp-timeago.js->' +
+          'third_party/timeagojs/timeago.js',
       '3p/polyfills.js->third_party/babel/custom-babel-helpers.js',
       'src/sanitizer.js->third_party/caja/html-sanitizer.js',
       'extensions/amp-viz-vega/**->third_party/vega/vega.js',
       'extensions/amp-viz-vega/**->third_party/d3/d3.js',
       'src/dom.js->third_party/css-escape/css-escape.js',
       'src/shadow-embed.js->third_party/webcomponentsjs/ShadowCSS.js',
+      'third_party/timeagojs/timeago.js->third_party/timeagojs/locales/ar.js',
```

Comment:
I think the important thing would be for them to be able to depend on itself, but not others.